linux_os/guide/services/ssh/ssh_server/sshd_use_strong_macs/oval/shared.xml

```diff
@@ -1 +1,70 @@
-{{{ oval_sshd_config(parameter="MACs", value="((hmac-sha2-512-etm@openssh\.com|hmac-sha2-256-etm@openssh\.com|umac-128-etm@openssh\.com|hmac-sha2-512|hmac-sha2-256|hmac-ripemd160),?)+") }}}
+<def-group>
+  <definition class="compliance" id="sshd_use_strong_macs" version="1">
+    {{{ oval_metadata("Ensure only strong MAC algorithms are used") }}}
+    <criteria operator="AND">
+      <criteria comment="SSH is configured correctly or is not installed"
+      operator="OR">
+        <criteria comment="sshd is not installed" operator="AND">
+          <extend_definition comment="sshd is not required or requirement is unset"
+          definition_ref="sshd_not_required_or_unset" />
+          {{% if product in ['opensuse', 'sle12', 'sle15'] %}}
+          <extend_definition comment="rpm package openssh removed"
+          definition_ref="package_openssh_removed" />
+          {{% else %}}
+          <extend_definition comment="rpm package openssh-server removed"
```

Comment:
could you please remove 'rpm' from the comment, different distros use different packaging schemas.